### ISSUE TITLE
setMaxX and setMinX not working on ridgeline plots

### ISSUE THREAD

**Issue by TheWildParamecium** [neuhausi/canvasXpress#130]
Good afternoon. I am sorry for the sudden issue again.

This time I found a weird behaviour in ridgeline plots (https://canvasxpress.org/examples/ridge-line-1.html# ) that prevents the user from setting custom max and min X axis values. The changes are not applied to the plot even if the values are set inside the config parameters. You will see, by looking at the setMaxX parameter inside the interactive parameter search (loupe icon) of the web example plot, that no value is being held inside setMaxX. I tried also using density plots(https://canvasxpress.org/examples/density-6.html#) and it does work in that case, so the problem seems to be limited to ridgeline plots.

I am leaving a toy example in jupyter notebook this time (as I found it easier to use at developing and debugging steps, even if we end up using the javascript CDN embedding in a HTML report in the end).

<code>
from canvasxpress.canvas import CanvasXpress
from canvasxpress.render.jupyter import CXNoteBook
cx = CanvasXpress(
   render_to = "ridgeline1",
   data = {
     "y": {
       "vars": [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30],
        "smps": ["assoc_value"],
       "data": [[4.8],[11.0],[46.03],[36.0],[4.05],[4.4],[16.3],[3.37],[3.17],[3.1],[45.6],[4.1],[4.8],[23.3],[3.3],[12.9],[9.1],[50.9],[5.94],[16.3],[60.8],[5.4],[7.15],[11.5],[11.4],[3.8],[5.7],[80.2],[8.9],[15.2]]
     },
    "x": {},
     "z": {
       "sample": ["OMIM","OMIM","OMIM","OMIM","OMIM","OMIM","OMIM","OMIM","OMIM","OMIM","ORPHA","ORPHA","ORPHA","ORPHA","ORPHA","ORPHA","ORPHA","ORPHA","ORPHA","ORPHA","MONDO","MONDO","MONDO","MONDO","MONDO","MONDO","MONDO","MONDO","MONDO","MONDO"]
     },
   },
   config = {
     "background": "rgb(255,255,255)",
     "colorBy": "sample",
     "customizerSize": "xxsmall",
     "dataTableTransposed": False,
     "graphOrientation": "vertical",
     "graphType": "Scatter2D",
     "hideHistogram": "True",
     "plotBox": False,
     "plotBoxColor": "rgb(204,204,204)",
     "ridgeBy": "sample",
     "ridgelineScale": 2,
     "showFilledHistogramDensity": True,
     "showHistogramDensity": True,
     "showLegendBorder": True,
     "title": "Association values distributions",
     "xAxis": ["Bin"],
     "setMaxX": 20,
     "setMinX": 5,
     "xAxisExact": True,
     "xAxisTickRightShow": False,
     "xAxisTitle": "x_axis",
     "yAxis": ["OMIM", "ORPHA", "MONDO"],
     "yAxisTickTopShow": False
   },
   after_render = [
     [
       "createHistogram",
       ["sample",None,None],
       {},
       1682676207842,
       True
     ],
     [
       "clickGraphMaxMin",
       [False,False,True],
       {},
       1682676265589
     ]
   ],
   width = 827,
   height = 410
 )
 
display = CXNoteBook(cx)
display.render()
</code>

<a href="https://ibb.co/ykFyQd1"><img src="https://i.ibb.co/VCYSqJZ/c-X-ridgeline1.png" alt="c-X-ridgeline1" border="0"></a>

Thank you so much for your help.

**Comment by neuhausi** [comment]
I will look into it.

**Comment by neuhausi** [comment]
Fixed in Version 43.6

![cX-ridgeline1](https://user-images.githubusercontent.com/254923/235558103-443781d7-9c3a-47de-afb5-d85b702bf542.png)

Here is the config for the graph (ridge-line-1.html)

```
{
     "colorBy": "Species",
     "graphType": "Scatter2D",
     "hideHistogram": true,
     "histogramBins": 20,
     "ridgeBy": "Species",
     "ridgelineScale": 2,
     "setMaxX": 10,
     "setMinX": 0,
     "showFilledHistogramDensity": true,
     "showHistogramDensity": true
}
```

**Comment by TheWildParamecium** [comment]
I can confirm, both with jupyter notebook and also with the javascript library through the CDN in an HTML report, that the functionally is working correctly now. Thank you so much for your help.
